### PR TITLE
feat(segmented-control): add SegmentedControl component

### PR DESCRIPTION
**Pull request by mediremi**
Closes https://jira.dhis2.org/browse/TECH-410

Adds a [`SegmentedControl`](https://github.com/dhis2/design-system/blob/master/molecules/segmented-control.md) component used to select between options that relate to another area of content.

image

### CODE REVIEW

**Comment by cooper-joe** [comment]
Nice, looks good!

Is there any way to avoid the double border between _segments_? Perhaps a `border-right:none` combined with `:first` and `:last` selectors. Screenshot of the double-between-border:
image

**Comment by mediremi** [comment]
I've fixed the double border between unselected segments, but should I also apply a fix for the double border between a selected segment and an unselected segment?

| Current behaviour | Possible fix |
| ------------------------ | --------------- |
| image | image |

**Comment by cooper-joe** [comment]
> should I also apply a fix for the double border between a selected segment and an unselected segment?

Yes, I think this makes the most visual sense. Thanks 👍

File: components/segmented-control/src/segmented-control.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+SegmentedControl.propTypes = {
+    options: PropTypes.arrayOf(
+        PropTypes.shape({
+            label: PropTypes.string.isRequired,
+            value: PropTypes.string.isRequired,
+            disabled: PropTypes.bool,
+        })
+    ).isRequired,
+    selected: PropTypes.string.isRequired,
+    onChange: PropTypes.func.isRequired,
```
[the remark this answers is not in the document]
Comment (mediremi) [reply components/segmented-control/src/segmented-control.js:123]: Thanks!

**Comment by mediremi** [comment]
@cooper-joe fixed :+1:

**Review by cooper-joe — APPROVED** [review]

**Pull request merged**